src/commands/messageForwarding.js

```diff
@@ -107,4 +108,41 @@ function forwardMessage(chesster, adminSlack) {
         });
     };
 }
+
+function refreshLeague(chesster, adminSlack) {
+    return function(bot, message) {
+        if (!_.isEqual(message.channel, adminSlack.config.messageForwarding.channel)) {
+            return;
+        }
+        var commandDescription = [
+            "refresh",
+            "{text:type}",
+            "{text:league}"
+        ];
+        return commands.tokenize(message.text, commandDescription).then(function(parameters){
+            var type = parameters['type'];
+            var leagueName = parameters['league'];
+            var _league = league.getLeague(bot, leagueName, chesster.config);
+            
+            if (!_league) {
+                winston.error("No matching league: {}".format(leagueName));
+                return;
+            }
+
+            if (type === "pairings") {
+                _league.refreshCurrentRoundSchedules().then(function(pairings) {
+                    winston.info("Found " + pairings.length + " pairings for " + _league.options.name + " (manual refresh)");
+                }).catch(function(error) {
+                    winston.error("{}: Error refreshing pairings: {}".format(_league.options.name, JSON.stringify(error)));
+                });
+            }
```

Comment:
It probably doesn't really matter - but we could log what the other `type` is here in case something gets messed up.